models/stg_quickbooks__deposit.sql

```diff
@@ -44,6 +44,7 @@ final as (
         currency_id,
         cast(department_id as {{ dbt.type_string() }}) as department_id,
         total_amount,
+        home_total_amount,
```

Comment:
Please be sure to add this to the [src](https://github.com/fivetran/dbt_quickbooks_source/blob/f9fb58596c28c395a4d45ac1e42fb0f3e2d694b6/models/src_quickbooks.yml#L363-L384) and [stg.yml](https://github.com/fivetran/dbt_quickbooks_source/blob/f9fb58596c28c395a4d45ac1e42fb0f3e2d694b6/models/stg_quickbooks.yml#L458-L487) files and regen the docs.